build/README.md

```diff
@@ -1,57 +1,75 @@
-# kprobe build setup
+# Building kprobe
 
-kprobe is distributed as a static binary linked with musl libc which
-requires a proper cross compilation environment. A reproducible
-build environment can be created using docker:
+Building kprobe is honestly a mess at this point. The streamlined build
```

Comment:
This is a public repo. I'm just saying...

Reply:
Fair